frontend/src/components/user/ATSUserLinkModal.vue

```diff
@@ -0,0 +1,172 @@
+<script setup lang="ts">
+import axios from 'axios';
+import { ref } from 'vue';
+
+import { Spinner } from '@/components/layout';
+import { Button, Column, DataTable, Dialog, InputText, useToast } from '@/lib/primevue';
+import { atsService } from '@/services';
+
+import type { Ref } from 'vue';
+import type { ATSUser } from '@/types';
+
+// Emits
+const emit = defineEmits(['atsUserLink:link']);
+
+// State
+const loading: Ref<boolean> = ref(false);
+const firstName: Ref<string> = ref('');
+const lastName: Ref<string> = ref('');
+const atsClientNumber: Ref<string> = ref('');
+const phone: Ref<string> = ref('');
+const email: Ref<string> = ref('');
+const selectedUser: Ref<ATSUser | undefined> = ref(undefined);
+const users: Ref<Array<ATSUser>> = ref([]);
+const visible = defineModel<boolean>('visible');
+
+// Actions
+
+const toast = useToast();
+
+async function searchATSUsers() {
+  selectedUser.value = undefined;
+  try {
+    loading.value = true;
+
+    const response = await atsService.searchATSUsers({
+      firstName: firstName.value,
+      lastName: lastName.value,
+      clientId: atsClientNumber.value,
+      phone: phone.value,
+      email: email.value
+    });
+
+    users.value = response.data.clients.map((client: any) => {
+      return {
+        atsClientNumber: client.clientId,
+        firstName: client.firstName,
+        lastName: client.surName,
+        email: client.address.email,
+        phone: client.address.primaryPhone ?? client.address.secondaryPhone,
+        address: `${client.address.addressLine1}, ${client.address.addressLine2}`
+      };
+    });
+  } catch (error) {
+    if (!axios.isCancel(error)) toast.error('Error searching for users ' + error);
+  } finally {
+    loading.value = false;
+  }
+}
+</script>
+
+<template>
+  <Dialog
+    v-model:visible="visible"
+    :draggable="false"
+    :modal="true"
+    class="app-info-dialog w-8"
+  >
+    <template #header>
+      <span class="p-dialog-title">Search ATS</span>
+    </template>
+
+    <div class="pt-1 mb-4 flex justify-content-between">
+      <div class="col-11.5 align-content-center">
```

Comment:
I don't believe half `col`s exist. They must be whole numbers. At least I don't see them in the https://primeflex.org/gridsystem docs